src/bitcoin/fee.rs

```diff
@@ -221,4 +221,18 @@ mod tests {
         let fee_rate = SatPerVByte::from_sat(100);
         assert_eq!(format!("{}", fee_rate), "100 satoshi/vByte".to_string());
     }
+
+    #[test]
+    fn serialize_fee_strat_in_yaml() {
+        let fee_strat = SatPerVByte::from_sat(10);
+        let s = serde_yaml::to_string(&fee_strat).expect("Encode fee strategy in yaml");
+        assert_eq!("---\n10\n", s);
```

Comment:
I'd prefer the unit as part of the serialization, but I see that would break the rust-bitcoin amount serde serialization. Can you entertain the possibility of serializing it as a string?